feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md

```diff
@@ -0,0 +1,116 @@
+# RT-5.7 Aggregate Not Viable All
```

Comment:
I think the summary should be something like: "Test forwarding-viable with LAG and routing"?